samples/cpp/tutorial_code/HighGUI/AddingImagesTrackbar.cpp

```diff
@@ -4,9 +4,7 @@
  * @author OpenCV team
  */
 
-#include "opencv2/imgcodecs.hpp"
```

Comment:
Is there a reason why you are removing imgcodecs? Imgproc.hpp is not implicitly including it.